hardware/raspberrypi/power/README.md

```diff
@@ -26,6 +26,14 @@ The power requirements of the Raspberry Pi increase as you make use of the vario
 
 If you need to connect a USB device that will take the power requirements above the values specified in the table above, then you must connect it to an externally-powered USB hub.
 
+## Power Supply Warnings
+
+On all models of Raspberry Pi since the Raspberry Pi B+ (2014) there is voltage detection circuitry that will detect if the supply voltage drops below 4.63v (+/- 5%). This will result in a [warning icon](../../../configuration/warning-icons.md) being displayed on all attached displays and an entry being being placed in the kernel log. 
+
+If you are seeing warnings, you should improve the power supply arrangement, as low power can cause problems with corruption of SD cards, or erratic behaviour of the Pi itself; for example, unexplained crashes. 
```

Comment:
"improve the power supply arrangement" sounds a bit too fluffy IMHO :wink:  Maybe something like "use a higher-current power supply" or similar?

Reply:
I did wonder about that - but it could be the cable, not the PS itself, so needs to cover that as well.

Reply:
How about "power supply chain" rather than arrangement?

Reply:
I'm not sure people will understand what "power supply chain" means? :chains:  Does "you should use a better power supply and/or power cable" work? :man_shrugging: